redash/tasks/queries/execution.py

```diff
@@ -26,75 +27,6 @@ def _unlock(query_hash, data_source_id):
     redis_connection.delete(_job_lock_id(query_hash, data_source_id))
 
 
-class QueryTask(object):
```

Comment:
:wave: good bye old friend. This class survived since the very first iterations of Redash (it was actually named `Job` back then), when we had my half-assed background job implementation and Tornado as the web server.